client/package.json

```diff
@@ -832,6 +823,7 @@
   },
   "dependencies": {
     "find": "^0.2.7",
+    "language-server-bitbake": "^2.2.0",
```

Comment:
The change breaks our debug configurations (launch.json). Since launching the client starts the server from npm instead of the local one. Also it looks like the server is running, but on VSCode most features seem broken? like references, hover documentation, ...